r-pkg/R/get_fields.R

```diff
@@ -53,14 +83,14 @@ get_fields <- function(es_host
         log_warn(sprintf(
             paste0(
                 "You are running Elasticsearch version '%s.x'. _all is not supported in this version."
-                , " Pulling all indices with 'POST /_cat/indices' for you."  # nolint[non_portable_path]
+                , " Pulling all indices with 'GET /_cat/indices' for you."  # nolint[non_portable_path]
```

Comment:
Another long-lived bug! This has been a `GET` request since the moment it was introduced back in 2019 😬 

ref: #161